<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QWidget" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>window</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(255,255,255);</string>
  </property>
  <widget class="QPushButton" name="push_play">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/play.png</normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/play.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>551</width>
     <height>24</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>Жанр</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Жанр</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Класика</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Джаз</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Поп</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Блюз</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Елктронна музика</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Рок</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Реп</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="push_pause">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/pause.png</normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/pause.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="push_skip_prev">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/skipfor.png</normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/skipfor.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="push_stop">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/stop.png</normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/stop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="push_skip_next">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/skipback.png</normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/skipback.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/add.jpg</normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/add.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/delete.jpg</normaloff>../build-audio-Desktop_Qt_5_8_0_MinGW_32bit-Debug/delete.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QTableView" name="playlistView">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>551</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="current_track">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="search">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>130</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_search">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>130</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 140, 255) ;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: white;</string>
   </property>
   <property name="text">
    <string>Знайти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_show">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 140, 255) ;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: white;</string>
   </property>
   <property name="text">
    <string>Показати всі</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_add">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 140, 255) ;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: white;</string>
   </property>
   <property name="text">
    <string>Додати плейлист</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_del">
   <property name="geometry">
    <rect>
     <x>659</x>
     <y>10</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 140, 255) ;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: white;</string>
   </property>
   <property name="text">
    <string>Видалити плейлист</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Плейлісти</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="push_adm">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 140, 255) ;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: white;</string>
   </property>
   <property name="text">
    <string>Створити</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="table_name">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Введіть назву плейлиста&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 140, 255) ;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: white;</string>
   </property>
   <property name="text">
    <string>Видалити</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_clear">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(10, 140, 255) ;
     border-style: outset;
     border-width: 2px;
     border-radius: 10px;
     border-color: white;</string>
   </property>
   <property name="text">
    <string>Очистити </string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
